modules/node/src/transfer/transfer.service.ts

```diff
@@ -104,10 +104,15 @@ export class TransferService {
   }
 
   // @hunter -- this should be pulling from the transfer view right?
+  // HH - I'm not sure? I have a good handle on db structure but not how the system typically interacts with it
   async getLinkedTransferByPaymentId(paymentId: string): Promise<LinkedTransfer | undefined> {
     return await this.linkedTransferRepository.findByPaymentId(paymentId);
   }
 
+  async getLinkedTransfersByUserPublicIdentifier(publicIdentifier: string): Promise<LinkedTransfer[]> {
+    return await this.linkedTransferRepository.findAllByRecipient(publicIdentifier);
+  }
```

Comment:
yeah, the function should definitely be renamed if it's really only looking it up by the `recipient` public identifier. ideally the function named `getLinkedTransfersByUserPublicIdentifier` would return ALL linked transfers with that user (whether they are recipient or sender)